frontend/src/components/AddNewTransactionNotificationForm.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from "react";
+import styles from "./Addnew.module.css";
+import axios from "axios";
+
+const SelectPerson = ({ persons }) => (
+  <select>
+    <option value="">Valitse henkilö</option>
+    {persons.map((person) => {
+      const { id, firstname, lastname } = person;
+      return (
+        <option key={id} value={id}>
+          {firstname} {lastname}
+        </option>
+      );
+    })}
+  </select>
+);
+
+const AddNewTransactionForm = ({ persons }) => {
+  const [sharesStart, setSharesStart] = useState("");
+  const [sharesEnd, setSharesEnd] = useState("");
+  const [pricePerShare, setPricePerShare] = useState("");
+  const [shareCount, setShareCount] = useState(0);
+  const [totalPrice, setTotalPrice] = useState(0);
```

Comment:
Can't we make an object state like AddNewPerson instead of having multiple states here?

Reply:
yes of course. My ticket was just to work on the uusi osakkeen omistaja side. There's another ticket for this lisää uusi ilmoitus osakemyynnistä.